public/src/components/amounts/AmountsVariantEditorRowInput.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { Button, makeStyles, TextField, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles(({ spacing }: Theme) => ({
+  textField: {
+    width: '120px',
+  },
+  button: {
+    minWidth: '20px',
+    marginLeft: spacing(2),
+  },
+}));
+
+interface AmountsVariantEditorRowInputProps {
+  amounts: number[];
+  addAmount: (amount: number) => void;
+}
+
+export const AmountsVariantEditorRowInput: React.FC<AmountsVariantEditorRowInputProps> = ({
+  amounts,
+  addAmount,
+}: AmountsVariantEditorRowInputProps) => {
+  const [currentValue, setCurrentValue] = useState<number | undefined>();
+  const [currentError, setCurrentError] = useState('');
+
+  const onSubmit = () => {
+    if (!currentError && currentValue != null && currentValue) {
```

Comment:
Does the `&& currentValue` here make the `currentValue != null` redundant?

Reply:
Fixed